File: 은지/LV2/N개의 최소공배수.js
function solution(arr) {
  while (arr.length > 1) {
    var a = arr.shift()
    var b = arr.shift()
    var v = Math.min(a, b)
    var gcd = v
    for (let x=1; x<=v; x++) {
      if (a%x === 0 && b%x === 0) {
        gcd = x
      }
    }
    arr.push(parseInt((a*b)/gcd))
  }
  return arr[0];
}

File: 은지/LV2/N개의 최소공배수.py
def solution(arr):
    while len(arr) > 1:
        a = arr.pop(0)
        b = arr.pop(0)
        v = min(a, b)
        gcd = v
        for x in range(1, v+1):
            if a % x == 0 and b % x == 0:
                gcd = x
        arr.append((a*b)//gcd)
    return arr[0]